metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfValidator.java

```diff
@@ -42,6 +42,7 @@ private static boolean validateNotInFuture(AbstractEdtfDate edtfDate) {
     return validateIntervalNotInFuture((IntervalEdtfDate) edtfDate);
   }
 
+
   // TODO: 20/07/2022 It checks interval of date parts but not time parts??
```

Comment:
This is valid. It has been decided that the time part is not relevant during validation.
The Instant date contains the time part though, which might not be used elsewhere.